client/package.json

```diff
@@ -20,8 +20,13 @@
     "styled-components": "^5.3.3"
```

Comment:
I'd suggest adding a `type-check` script to `package.json`: `"type-check": "tsc --noEmit"`

This gives us a way to manually check the types and can be reused later when we set up GitHub Actions.

Reply:
Added